workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/IReaderInternal.cs

```diff
@@ -1,4 +1,7 @@
-﻿using Improbable.Worker;
+﻿using System.Runtime.CompilerServices;
+using Improbable.Worker;
+
+[assembly: InternalsVisibleTo("Improbable.Gdk.Core.EditmodeTests")]
```

Comment:
What is this for? Is this here because `Improbable.Gdk.Core.MonoBehaviours` is otherwise not visible in the editmode tests?

Reply:
This is because tests such as `AuthorityChanged_gets_triggered_when_the_reader_receives_an_authority_change` need to use the IReaderInternal interface, but we do not want non-core modules to see it